packages/create-novu-app/templates/app-react-email/ts/app/api/novu/route.ts

```diff
@@ -1,4 +1,6 @@
 import { serve } from "@novu/framework/next";
-import { myWorkflow } from "../../novu/workflows";
+import { welcomeOnboardingEmail } from "../../novu/workflows";
 
-export const { GET, POST, OPTIONS } = serve({ workflows: [myWorkflow] });
+export const { GET, POST, OPTIONS } = serve({
+  workflows: [welcomeOnboardingEmail],
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Consider adding a comment here that this is where you can add new workflows (per KMac's question / confusion during the demo)‏

Reply:
done